visivo/models/targets/sqlalchemy_target.py

```diff
@@ -14,7 +12,10 @@ class SqlalchemyTarget(Target, ABC):
     def get_dialect(self):
         raise NotImplementedError(f"No dialect method implemented for {self.type}")
 
-    def read_sql(self, query: str) -> DataFrame:
+    def read_sql(self, query: str):
+        from pandas import DataFrame
```

Comment:
We could probably refactor this to not use pandas at all, if that would help.

Reply:
I think right now it is fine, we can look into other solutions if we need in the future.